### PR TITLE
chore: [#176981159,#177092221] Make credit card onboarding compliant with 3DS2 and track onboarding events

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR add the new payment step (in webview) and remove the old one.

## List of changes proposed in this pull request
Now after the credit card is added to a temporary wallet a web view is shown where is injected a form with these values:

- idWallet
- securityCode
- sessionToken
- language

After the 2 cent payment in the webview succeed a thank you page is shown.
Instead if the webview does not succeed a ko page is shown.

**What have been removed**

- Removed the credit card verification step, handled inside the webview
- Removed the 3ds checkout step, handled inside the webview

## How to test
Make sure to have the last commit of io-dev-server (master branch)

### CODE REVIEW

File: locales/it/index.yml [hunk 1]
```diff
@@ -1296,6 +1296,11 @@ wallet:
       success:
         title: "Grazie, l’operazione è stata eseguita con successo!"
         description: "Abbiamo inviato una ricevuta al tuo indirizzo email. Puoi trovarla anche nel tuo Portafoglio."
+  abortWebView:
+    title: Vuoi interrompere l'aggiunta della carta di credito in corso?
```
Comment (thisisjp) [inline locales/it/index.yml:1300]:
```suggestion
    title: Vuoi uscire dall'operazione in corso?
```

File: locales/it/index.yml [hunk 2]
```diff
@@ -1296,6 +1296,11 @@ wallet:
...
+    confirm: Annulla l'aggiunta
```
Comment (thisisjp) [inline locales/it/index.yml:1301]:
```suggestion
    confirm: Sì, esci
```

File: locales/it/index.yml [hunk 3]
```diff
@@ -1296,6 +1296,11 @@ wallet:
...
+    cancel: Continua con l'aggiunta
```
Comment (thisisjp) [inline locales/it/index.yml:1302]:
```suggestion
    cancel: No, continua
```

File: locales/it/index.yml [hunk 4]
```diff
@@ -1296,6 +1296,11 @@ wallet:
...
+    success: L'aggiunta della carta di credito è stata annullata correttamente
```
Comment (thisisjp) [inline locales/it/index.yml:1303]: @debiff @Undermaken this is quite an interesting use case. If you abort the flow, it's very likely you're having issues with the 3DS codes. Maybe we can suggest users to get in touch with their bank. I'll create a story ;)

File: locales/en/index.yml [hunk 5]
```diff
@@ -1271,6 +1271,11 @@ wallet:
       success:
         title: "Thank you for your payment!"
         description: "We sent a receipt to your email address. You can also find it in your Wallet."
+  abortWebView:
+    title: Do you want to cancel the credit card onboarding in progress?
+    confirm: Cancel the onboarding
+    cancel: Continue with the onboarding
```
Comment (thisisjp) [inline locales/en/index.yml:1277]:
```suggestion
    cancel: Keep adding
```

File: locales/en/index.yml [hunk 6]
```diff
@@ -1271,6 +1271,11 @@ wallet:
...
+    success: The credit card onboarding was successfully canceled
```
Comment (thisisjp) [inline locales/en/index.yml:1278]:
```suggestion
    success: Problems while adding your card? Please contact your bank.
```

File: locales/it/index.yml (hunk 4 above)
Comment (thisisjp) [inline locales/it/index.yml:1303]:
```suggestion
    success: Problemi con l'aggiunta? Contatta la tua banca.
```

File: locales/en/index.yml [hunk 7]
```diff
@@ -1271,6 +1271,11 @@ wallet:
       success:
         title: "Thank you for your payment!"
         description: "We sent a receipt to your email address. You can also find it in your Wallet."
+  abortWebView:
+    title: Do you want to cancel the credit card onboarding in progress?
+    confirm: Cancel the onboarding
```
Comment (thisisjp) [inline locales/en/index.yml:1276]:
```suggestion
    confirm: Yes, quit
```

File: locales/en/index.yml [hunk 8]
```diff
@@ -1271,6 +1271,11 @@ wallet:
       success:
         title: "Thank you for your payment!"
         description: "We sent a receipt to your email address. You can also find it in your Wallet."
+  abortWebView:
+    title: Do you want to cancel the credit card onboarding in progress?
```
Comment (thisisjp) [inline locales/en/index.yml:1275]:
```suggestion
    title: Quit adding your card?
```

**Review by thisisjp — APPROVED** [review]

**Review by Undermaken — APPROVED** [review]

**Pull request merged**